src/components/App.tsx

```diff
@@ -2456,9 +2457,15 @@ export class App extends React.Component<any, AppState> {
         delta = MAX_STEP;
       }
       delta *= sign;
+      const newElements = { ...selectedElementIds };
+      Object.keys(newElements).forEach((key) => {
+        newElements[key] = false;
+      });
       this.setState(({ zoom }) => ({
         zoom: getNormalizedZoom(zoom - delta / 100),
+        selectedElementIds: newElements,
```

Comment:
Shouldn’t you just do selectedElementIds: {} ?

Reply:
You're right, I changed it !